executor/importer/table_import.go

```diff
@@ -147,6 +147,29 @@ func GetCachedKVStoreFrom(pdAddr string, tls *common.TLS) (tidbkv.Storage, error
 	return kvStore, nil
 }
 
+// GetRegionSplitSizeKeys gets the region split size and keys from PD.
+func GetRegionSplitSizeKeys(ctx context.Context) (regionSplitSize int64, regionSplitKeys int64, err error) {
+	tidbCfg := tidb.GetGlobalConfig()
+	hostPort := net.JoinHostPort("127.0.0.1", strconv.Itoa(int(tidbCfg.Status.StatusPort)))
```

Comment:
I see that the `hostPort` is used by `tls`, and in fact `tls` will overwrite the host by `WithHost` in `getSplitConfFromStore`. So maybe don't need to set the actual value.

Reply:
https://github.com/pingcap/tidb/pull/46998/commits/e25413d647f9ae7b6ac9a8991cd292a828523bab